components/Register.vue

```diff
@@ -183,6 +166,13 @@ export default {
     },
     maximizeModal() {
       return this.$q.screen.lt.md;
+    },
+    tos() {
+      const {vueWallet: {terms = ''}} = config;
+      if(terms.length === 0) {
+        return ['These are the terms of service.'];
```

Comment:
But "These are the terms of service." are not terms of service. ;-)  Maybe the default could be something that stands alone like "Please contact us for the terms of service."?

Reply:
I also assume at some point the terms will be HTML with headers and lists and such, so this line splitting paragraph method may have to go away.

Reply:
Updated in a7b29e2efdbfc2f78b0063d8dbc9aa3c8a15d2b4

> I also assume at some point the terms will be HTML with headers and lists and such, so this line splitting paragraph method may have to go away.

HTML is a good option. I could also see having a pre-prepared terms of service in PDF form could be desirable as well.